scripts/superpmi.py

```diff
@@ -452,18 +456,71 @@ def __collect_mc_files__(self, command, args):
             print_platform_specific_environment_vars(self.coreclr_args, "COMPlus_AltJit", "*")
             print_platform_specific_environment_vars(self.coreclr_args, "COMPlus_AltJitName", self.collection_shim_name)
             print("")
-            print("%s %s" % (command, " ".join(args)))
 
-            assert isinstance(command, str)
-            assert isinstance(args, list)
+            if self.command != None:
+                print("%s %s" % (command, " ".join(args)))
```

Comment:
Are `command` and `args` defined here without self?

Reply:
Technically yes because python because of dynamic binding. It should not be relied on will change thank you for catching this.